build/reports/tests/test/classes/CheckerFrameworkWarningResolverTest.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
```

Comment:
This file shouldn't be committed (nor should anything else under the `build` directory, which you should also add to the `.gitignore`)